### ISSUE TITLE
Docs: Shpinx Cheatsheet

### ISSUE THREAD

**Issue by ramirezfranciscof** [aiidateam/aiida-core#4033]
Move content from the [sphinx cheatsheet](https://aiida.readthedocs.io/projects/aiida-core/en/latest/developer_guide/tools/sphinx_cheatsheet.html) of the old documentation into the aiida-core wiki.

**Comment by ramirezfranciscof** [comment]
Question: should we maybe leave this inside the repo but separated from the documentation, so that it is not compiled and uploaded to the web, but that can be compiled to generate a webpage and you can open the file locally with your browser?

The issue with just adding it to the wiki is that I could not find a way to re-create the notes and autodocs using githubs wiki formats, so I could copy how to write them but would not be able to diplay what they look like.

Which option do you prefer?
(@csadorf @sphuber @giovannipizzi and any other interested party...)

**Comment by sphuber** [comment]
I personally really don't think we need the autogenerated examples. Honestly I don't even really know why we need to write our own cheatsheet. There are more complete and well maintained version available online. None of the things we add in the cheatsheet are AiiDA specific. The only thing we should write in our wiki is _our_ style guide rules and then we can of course put a link to a well written Sphinx manual or tutorial.

The same goes for the git cheatsheet that I think we have somewhere. Also there we should just have our git style requirements and just link to an existing cheatsheet or manual.

TLDR: I would say we just remove this entire page and add a link to good Sphinx manual in our wiki on Documentation style and guidelines

**Comment by csadorf** [comment]
I agree with @sphuber .

**Comment by ramirezfranciscof** [comment]
Ok. There already is a link to [the sphynx basics webpage](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) in the section of the wiki [for aiida documentation](https://github.com/aiidateam/aiida-core/wiki/Writing-documentation). I guess the official sphinx page should always be the most up to date resource. I will just remove this then (and I am also deleting the reference to it in the linked page of the wiki).

**Comment by sphuber** [comment]
Fixed in PR #4069
